src/odb/src/db/dbBox.cpp

```diff
@@ -741,12 +750,22 @@ dbTechLayer* dbBox::getTechLayer()
   return (dbTechLayer*) box->getTechLayer();
 }
 
+uint dbBox::getLayerMask()
+{
+  _dbBox* box = (_dbBox*) this;
+  if (getTechLayer() == nullptr) {
```

Comment:
Could you have a non-zero mask but no layer?  I'm wondering if this check is needed here or this should be enforced in dbBox::create

Reply:
looking at the DEF I don't think you can. I can remove that check since the `_layer_mask` gets initialized to 0 anyways. I'll look at the ::create and add checks there.